package.json

```diff
@@ -8,7 +8,7 @@
     "preview": "svelte-kit preview",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "test": "jest",
+    "test": "jest --env=jsdom",
```

Comment:
Is this command generally something you need for front end testing in jest?

Reply:
It is something I have typically had on my previous projects, and it resolved an error when trying to run the tests here.